WalletWasabi/Tor/Http/TorHttpClient.cs

```diff
@@ -46,7 +46,7 @@ public TorHttpClient(Func<Uri> baseUriAction, EndPoint? torSocks5EndPoint, bool
 		public static DateTimeOffset? TorDoesntWorkSince
 		{
 			get => TorDoesntWorkSinceBacking;
-			private set
+			set
```

Comment:
Hmm... Why?

Reply:
I have reverted it. It may / may not be a good idea. Let's leave it for the future to avoid complicating this PR.